softlayer/data_source_softlayer_image_template_test.go

```diff
@@ -0,0 +1,39 @@
+package softlayer
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccSoftLayerImageTemplateDataSource_Basic(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckSoftLayerImageTemplateDataSourceConfig_basic,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(
+						"data.softlayer_image_template.tfacc_img_tmpl",
+						"name",
+						"jumpbox",
+					),
+					resource.TestCheckResourceAttr(
+						"data.softlayer_image_template.tfacc_img_tmpl",
+						"id",
+						"988251",
+					),
```

Comment:
Suggestion: just check that some ID was stored, so that the test case is more portable.  e.g.:
 
`resource.TestMatchResourceAttr("data.softlayer_image_template.tfacc_img_tmpl", "id", regexp.MustCompile("^[0-9]+")),`